### PR TITLE
Turkish March 2016 Update

### PR DESCRIPTION
**Pull request by kyzn**
Several fixes for Turkish Language, syncing with CrowdIn. Cheers to all translators!

### CODE REVIEW

File: tr/SUMMARY.md [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
-* [Django ORM (Querysets - Sorgu setleri)](django_orm/README.md)
-* [Şablonlarda dinamik veri](dynamic_data_in_templates/README.md)
-* [Djago şablonları](django_templates/README.md)
-* [CSS - sayfanı güzelleştir](css/README.md)
-* [Template(Şablon) extending(devam ettirme)](template_extending/README.md)
-* [Uygulamanı genişlet](extend_your_application/README.md)
-* [Django formları](django_forms/README.md)
-* [Sırada ne var?](whats_next/README.md)
\ No newline at end of file
+* [Giriş][README.md]
```
Comment (tricoder42) [inline tr/SUMMARY.md]: Links needs to be in brackets, but not it square brackets, like so: 

```
[Link title](Link url)
```

File: tr/django_models/README.md [hunk 2]
```diff
@@ -1,52 +1,54 @@
...
+tirmala() 
+beslen(kedi_mamasi) 
+
+KediMamasi
+-------- 
+tat
+```
 
-Yani aslında ana fikir, gerçek nesneleri kod içinde özellikleriyle (`nesne özellikleri`) ve hareketleriyle (`metodlar`) tanımlamak.
+Yani aslında ana fikir, gerçek nesneleri kod içinde özellikleriyle (`nesne özellikleri`) ve hareketleriyle (`metodlar`) tanımlamak.).
```
Comment (tricoder42) [inline tr/django_models/README.md]: Extra bracket?

Comment (kyzn) [inline tr/django_models/README.md]: This is a bit CrowdIn's fault I guess. See the English sentence (original one). The last right parenthesis is not in the translatable region (it is added automatically). Turkish translation does not need it (not all languages have the same word order, obviously), so we cannot really remove that parenthesis in CrowdIn. I'll update the file on GitHub, but I suppose we need a better solution than this. (Also see German and Korean translators had the same issue. I'm going to raise the issue with translations@djangogirls, to see if they can solve it.)  

y

Comment (tricoder42) [inline tr/django_models/README.md]: I see. I contacted Crowdin support and we'll fix it once they respond.

Comment (kyzn) [inline tr/django_models/README.md]: Thanks!

**Pull request merged**